Src/DBProj/Tables/Products.sql

```diff
@@ -2,12 +2,15 @@
     [ProductID]   INT          IDENTITY (1, 1) NOT NULL,
     [Name]        VARCHAR (50) NOT NULL,
     [Price]       FLOAT (53)   NOT NULL,
-    [PictureName] VARCHAR (50) NOT NULL,
     [CreatedOn]   DATETIME     NOT NULL,
     [CreatedBy]   INT          NOT NULL,
     [ModifiedOn]  DATETIME     NULL,
     [ModifiedBy]  INT          NULL,
     [IsActive]    BIT          NOT NULL,
+    [PictureName1] VARCHAR(50) NOT NULL, 
```

Comment:
Should we not have a separate table to maintain images of a product?

Reply:
I think It will not make a big difference while we can cover it in the same table?.

Reply:
> I think It will not make a big difference while we can cover it in the same table?.

What if we want to store 5 pictures of a product in future? Having separate table will allow us n number of pictures against a product.

Reply:
ok sir

Reply:
should we create a new pull request after making changes in our branch?

Reply:
No you just need to make commits in same branch. All new commits will be visible in opened PR

Reply:
I have completed my work kindly check it